src/applications/vaos/appointment-list/components/RequestedStatusAlert.jsx

```diff
@@ -27,34 +44,28 @@ export default function RequestedStatusAlert({ appointment, facility }) {
   if (showConfirmMsg) {
     return (
       <InfoAlert backgroundOnly status={canceled ? 'error' : 'success'}>
-        {canceled && `${who || 'Facility'} canceled this appointment.`}
+        {canceled && (
+          <>
+            <strong>{who || 'Facility'} canceled this request. </strong>
```

Comment:
Would 'Facility' response be handled by the logic on line 40?

Reply:
@ptouchton You're absolutely right. Great catch!!